pkg/sql/pg_catalog.go

```diff
@@ -2648,6 +2648,7 @@ func addPGTypeRow(
 	case types.EnumFamily:
 		builtinPrefix = "enum_"
 		typType = typTypeEnum
+		fallthrough
```

Comment:
super nit: not sure i like `fallthrough` - easy to trip. maybe

```
default:
   if typ.Family() == tpyes.EnumFamily {
      //... stuff
   }
   typArray = x
```

Reply:
done